frontend/src/js/controllers/teamsCtrl.js

```diff
@@ -341,7 +341,8 @@
                 parameters.url = 'participants/participant_team/' + participantTeamId + '/invite';
                 parameters.method = 'POST';
                 parameters.data = {
-                    "email": result
+                    "email": result,
+                    "url": $state.href('web.challenge-host-teams', {}, {absolute: true})
```

Comment:
@isht3 please change the url here.
Need to replace `web.challenge-host-teams` with `web.teams` 

also you need to change 
`full_url = url + "invitation/" + unique_hash`
to 
`full_url = url + "participant-invitation/" + unique_hash`

[Source](https://github.com/Cloud-CV/EvalAI/pull/1489/files#diff-936afd091c6c99c1a02293abc3be8653R264)